_layouts/home.html

```diff
@@ -23,6 +23,17 @@ <h3>
         </h3>
         {%- if site.show_excerpts -%}
           {{ post.excerpt }}
+          {% capture content_words %}
```

Comment:
Using `{% capture %}` always results in a *string*. Wouldn't it be better if we used `{% assign .. %}` instead..?
```
{% assign content_length = post.content | number_of_words %}
{% assign excerpt_length = post.excerpt | number_of_words %}
...
```

Reply:
I figure content and excerpt are string, that's why I used `{% capture %}`, but I don't see why we shouldn't use `{% assign %}` instead. Let me update the PR.

Reply:
`content` and `excerpt` are strings, but the result from ` | number_of_words` isn't..

Reply:
Ah right!